modules/Microsoft.DBforMySQL/flexibleServers/.test/.scripts/Get-PairedRegion.ps1

```diff
@@ -0,0 +1,16 @@
+﻿param(
+    [string] $Location
```

Comment:
Please add synopsis (like in PR #2681)

Reply:
And now that the other PR (#2681) actually one more: Please move it into the shared scripts folder (`/modules/.shared/.scripts`)

Reply:
Yep I noticed that too.